data.sql

```diff
@@ -1,5 +1,5 @@
 /* Populate database with sample data. */
-
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO `vet_clinic`.`animals` (`name`, `date_of_birth`, `escape_attempts`, `neutered`, `weight_kg`) VALUES ('Agumon', '2020-02-03', '0', '1', '10.23');
+INSERT INTO `vet_clinic`.`animals` (`name`, `date_of_birth`, `escape_attempts`, `neutered`, `weight_kg`) VALUES ('Gabumon', '2018-11-15', '2', '1', '8');
+INSERT INTO `vet_clinic`.`animals` (`name`, `date_of_birth`, `escape_attempts`, `neutered`, `weight_kg`) VALUES ('Pikachu', '2021-01-07', '1', '0', '15.04');
+INSERT INTO `vet_clinic`.`animals` (`name`, `date_of_birth`, `escape_attempts`, `neutered`, `weight_kg`) VALUES ('Devimon', '2017-05-12', '5', '1', '11');
```

Comment:
- [ ] In the data.sql, kindly make the following changes:
- Please change the name of the table from _**vet_clinic.animals**_ to `animals`.
- Kindly remove all backticks when you insert data. 
- The data for `neutered` should be `true` or `false` and not `'0'` or `'1'`.

In the end, when you try to insert data, it should look like this:
`INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Agumon', '2020-02-03', '0', true, '10.23');
`.

Kindly implement these changes. 👍